x-pack/metricbeat/module/mssql/os/data.go

```diff
@@ -0,0 +1,80 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package os
+
+import (
+	"time"
+
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	schema = s.Schema{
+		// Returns a miscellaneous set of useful information about the computer,
+		// and about the resources available to and consumed by SQL Server.
+		"info": s.Object{
+			"cpu_ticks":                      c.Int("cpu_ticks"),
+			"ms_ticks":                       c.Int("ms_ticks"),
+			"cpu_count":                      c.Int("cpu_count"),
+			"hyperthread_ratio":              c.Int("hyperthread_ratio"),
+			"physical_memory_in_bytes":       c.Int("physical_memory_in_bytes"),
+			"physical_memory_kb":             c.Int("physical_memory_kb"),
+			"virtual_memory_in_bytes":        c.Int("virtual_memory_in_bytes"),
+			"virtual_memory_kb":              c.Int("virtual_memory_kb"),
```

Comment:
Not sure what to do with units, but we should use same format, like:
```
memory.physical.bytes
memory.physical.kbytes
memory.virtual.bytes
memory.virtual.kbytes
```

Or just collect only bytes.

Reply:
Oh, I didn't know we have some "common" format for this. I'll change it rather than doing operations over them, specially when users could potentially know this fields already or could try to reason the nature of the metricset by consulting the documentation of MS

Reply:
We don't have a common format for this, I only proposed to use at least the same format here, i.e. not using "in_bytes" in some metrics, and "kb" in others.

Reply:
Do you have any preference? Because I don't have any :smile:

Reply:
I think that collecting bytes should be enough...